### PR TITLE
[OCPI] Show evse_id as charging station name for charging station managed by ocpi

### PR DESCRIPTION
**Pull request by ellavas**

### CODE REVIEW

File: src/app/pages/charging-stations/list/charging-stations-list-table-data-source.ts [hunk 1]
```diff
@@ -133,6 +133,9 @@ export class ChargingStationsListTableDataSource extends TableDataSource<Chargin
         class: 'col-30p',
         sorted: true,
         direction: 'asc',
+        formatter: (id: string, chargingStation: ChargingStation) => chargingStation.ocpiData
+          && chargingStation.ocpiData.evse && chargingStation.ocpiData.evse.evse_id ?
+          chargingStation.ocpiData.evse.evse_id : id,
```
Comment (LucasBrazi06) [inline src/app/pages/charging-stations/list/charging-stations-list-table-data-source.ts:138]: Everything is based on charging station id like filters, ...
If you change the display id of the charging station, it has to be done everywhere in the app.
What do you think?

Comment (LucasBrazi06) [reply src/app/pages/charging-stations/list/charging-stations-list-table-data-source.ts:138]: @ellavas Any news on this one ?

Comment (ellavas) [reply src/app/pages/charging-stations/list/charging-stations-list-table-data-source.ts:138]: Absolutely, it must be done everywhere!

Comment (LucasBrazi06) [reply src/app/pages/charging-stations/list/charging-stations-list-table-data-source.ts:138]: > Absolutely, it must be done everywhere!

Ok, I'll wait for your change.

Comment (LucasBrazi06) [reply src/app/pages/charging-stations/list/charging-stations-list-table-data-source.ts:138]: What is it displayed when evse_id is used versus id?

Comment (ellavas) [reply src/app/pages/charging-stations/list/charging-stations-list-table-data-source.ts:138]: one example: 
=> id = 22740-61273, evse_id => FR*SLF*ESAP*DEV*OCPI*PUBLIC*2*1

This is the more readable field in ocpi data.

Comment (LucasBrazi06) [reply src/app/pages/charging-stations/list/charging-stations-list-table-data-source.ts:138]: We keep the id as it is.

**Pull request closed without merging**